README.md

```diff
@@ -412,6 +413,35 @@ task :foo do
 end
 ```
 
+### Logger
+
+If you want to convert your log messages to Airbrake errors, you can use our
+integration with Ruby's `Logger` class from stdlib. All you need to do is to
+make sure that the integration is required (if the Airbrake is loaded after
+`Logger`, then it is already integrated):
+
+```ruby
+require 'airbrake/logger/logger_ext'
+```
+
+To start using the integration create a new Logger and assign an Airbrake
+notifier to it:
+
+```ruby
+logger = Logger.new(STDOUT)
+
+# New method.
+# We assign the default notifier here.
+logger.airbrake = Airbrake[:default]
```

Comment:
I would expect that `logger.airbrake == Airbrake[:default]` be default if you import `airbrake/logger/logger_ext`.